### DISCUSSION TITLE
Plan for 50K page views

### DISCUSSION THREAD

**Discussion by andrewholgate** [plausible/analytics#1937] [Anything Else]
The step up from the cheapest plan (10K page views) to the next plan is a dramatic 10 fold increase in usage (100K). The second biggest step up is only 2.5 times increase in page views (200K to 500K).

It just feels like there needs to be a plan in between, like 50K, in order to cater for project that are small and growing but are not yet medium sized yet.

![Screenshot_20220530-190657](https://user-images.githubusercontent.com/3862180/171036584-81059fad-c3f6-4f3b-a683-4f329832de42.png)

**Comment by metmarkosaric** [comment]
thanks for the feedback @andrewholgate! appreciate it and will keep it in mind for next time we review pricing.
